build/pause/Dockerfile.Rhel

```diff
@@ -1,13 +1,13 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.15 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.15 AS builder
 WORKDIR /go/src/github.com/openshift/kubernetes/build/pause
 COPY . .
 RUN dnf install -y gcc glibc-static && \
-    mkdir -p bin && \
-    gcc -Os -Wall -Werror -static -o bin/pause ./linux/pause.c
+  mkdir -p bin && \
+  gcc -Os -Wall -Werror -static -o bin/pause ./linux/pause.c
 
 FROM registry.ci.openshift.org/ocp/4.15:base
```

Comment:
This base image is RHEL 8, you might want to use `base-rhel9` instead.

Reply:
Fixed! Thanks!